website/docs/r/container_app.html.markdown

```diff
@@ -0,0 +1,337 @@
+* `container_app_environment_id` - (Required) The ID of the Container App Environment to host this Container App. Changing this forces a new resource to be created.
+
+* `name` - (Required) The name for this Container App. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the Container App Environment is to be created. Changing this forces a new resource to be created.
+
+* `revision_mode` - (Required) The revisions operational mode for the Container App. Possible values include `Single` and `Multiple`.
+
+* `template` - (Required) A `template` block as detailed below.
+
+---
+
+* `dapr` - (Optional) A `dapr` block as detailed below.
+
+* `ingress` - (Optional) An `ingress` block as detailed below.
+
+* `registry` - (Optional) A `registry` block as detailed below.
+
+* `secret` - (Optional) A `secret` block as detailed below.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+---
+
+A `secret` block supports the following:
+
+* `name` - (Required) The Secret name.
+
+* `value` - (Required) The value for this secret.
+
+!> **Note:** Secrets cannot be removed from the service once added, attempting to do so will result in an error. Their values may be zeroed, i.e. set to `""`, but the named secret must persist. This is due to a technical limitation on the service which causes the service to become unmanageable. See [this issue](https://github.com/microsoft/azure-container-apps/issues/395) for more details.
+
+---
+
+A `template` block supports the following:
+
+* `container` - (Required) A `container` block as detailed below.
+
+* `max_replicas` - (Optional) The maximum number of replicas for this container.
+
+* `min_replicas` - (Optional) The minimum number of replicas for this container.
+
+* `revision_suffix` - (Optional) The suffix for the revision. This value must be unique for the lifetime of the Resource. If omitted the service will use a hash function to create one.
+
+* `volume` - (Optional) A `volume` block as detailed below.
+
+---
+
+A `volume` block supports the following:
+
+* `name` - (Required) The name of the volume.
+
+* `storage_name` - (Optional) The name of the `AzureFile` storage.
+
+* `storage_type` - (Optional) The type of storage volume. Possible values include `AzureFile` and `EmptyDir`. Defaults to `EmptyDir`.
+
+---
+
+A `container` block supports the following:
+
+* `args` - (Optional) A list of args to pass to the container.
+
+* `command` - (Optional) A command to pass to the container to override the default. This is provided as a list of command line elements without spaces.
+
+* `cpu` - (Required) The amount of vCPU to allocate to the container. Possible values include `0.25`, `0.5`, `0.75`, `1.0`, `1.25`, `1.5`, `1.75`, and `2.0`. **NOTE:** `cpu` and `memory` must be specified in `0.25'/'0.5Gi` combination increments. e.g. `1.0` / `2.0` or `0.5` / `1.0`
+
+* `env` - (Optional) An `env` block as detailed below.
```

Comment:
expand?
```suggestion
* `environment` - (Optional) An `env` block as detailed below.
```

Reply:
This is specific Container/Docker parlance for this, so I think we should keep it the same here to help avoid confusion for users?